html-css/week-03/lesson.md

```diff
@@ -193,14 +193,15 @@ A checkbox allows the user to say yes or no to an option. With a checkbox, the u
 
 ---
 
-Checkboxes that are related to each other can be grouped together in a `<fieldset>`. A group of fields is described by a `<legend>`.
+Checkboxes that are related to each other can be grouped together in a `<fieldset>`. A group of fields is described by a `<legend>`. As you can see a fieldset (similar to a field) needs to be inside a form.
```

Comment:
I'd like to leave this out. The code example demonstrates this in a clearer way than text anyway, and in my view it helps if the explanation bits are as lean and focused as possible.

Reply:
I would leave it in... the students are still new to coding so for some of them just seeing it in code might not be that clear. I think having a short extra sentence in there explaining it in text, takes everyone along the journey also they one who aren't that familiar with spotting differences in code

Reply:
I guess this makes me more concerned about inconsistency. All the elements we cover in this lesson need to be inside of a form field, so I wonder if it's confusing to say it for fieldsets but not other fields.

Maybe we need to cover this more explicitly in the lesson. I'm thinking maybe once at the end of The HTML Form section, once at the end of Fields, once at the end of Labels and once at the end of Buttons?